.github/workflows/build_pods.yaml

```diff
@@ -0,0 +1,21 @@
+name: Build Pods
+on:
+  repository_dispatch:
+    types:
+    - build
+jobs:
+  build_pods:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: ManageIQ/manageiq-pods
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version-file: 'manageiq-operator/go.mod'
+    - name: Docker login
+      if: ${{ github.event_name != 'pull_request' }}
```

Comment:
This will still trigger on branch forks, which we probably don't want?

Reply:
Done